aten/src/THNN/README.md

```diff
@@ -1,27 +0,0 @@
-# THNN
```

Comment:
can we move this to THCUNN?  Since that will still be around (and doesn't have a corresponding doc, would be nice not to use this).

Reply:
done.